web/webplatform/VERSION

```diff
@@ -1 +1 @@
-2.29.3
+2.37.2 dfc6f412248826db2a070e9c5bbece778a69185224636deadc084d19b5fbc7ab
```

Comment:
The point of a checksum is that no one is allowed to specify the expected value other than the trusted source:

You download the expected checksum, and compare it to the actual checksum of the versioned tar you have/downloaded - and if they match you know the tar file wasn't tampered.

If you don't download the checksum, whoever tampers with the local tar file can also tamper with the version file.

Reply:
i'm not sure we're on the same page. a trusted source updates the VERSION file, that way you cannot hijack the link or whatever. This change also checks against the current file even if not download happened btw.

Reply:
talked to @tebeka , opined that this is the way for extra paranoids, so we're good.